### PR TITLE
Pull latest changes from opendata-stuttgart

### PR DESCRIPTION
**Pull request by DavidLemayian**
Pull latest changes from opendata-stuttgart/feinstaub-map.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,11 @@
...
     <link rel="shortcut icon" href="/static/img/favicon.ico">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,100,300,500,700,900,400italic,100italic,300italic,700italic,500italic,900italic" />
-    <link href='https://fonts.googleapis.com/css?family=Roboto+Mono:700,400,100,300' rel='stylesheet' type='text/css'>
-    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,700" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!-- proxy: https://fonts.googleapis.com/ to https://maps.luftdaten.info/fonts/ -->
+    <link rel="stylesheet" href="https://maps.luftdaten.info/fonts/css?family=Roboto:400,100,300,500,700,900,400italic,100italic,300italic,700italic,500italic,900italic" />
+    <link href='https://maps.luftdaten.info/fonts/css?family=Roboto+Mono:700,400,100,300' rel='stylesheet' type='text/css'>
+    <link href="https://maps.luftdaten.info/fonts/css?family=Roboto+Slab:400,700" rel="stylesheet">
+    <link href="https://maps.luftdaten.info/fonts/icon?family=Material+Icons" rel="stylesheet">
```
Comment (DavidLemayian) [inline index.html:11]: @chegejames Any idea why/how they're doing this proxy even though hosted on Github Pages?

File: src/map.vue [hunk 2]
```diff
@@ -16,7 +16,8 @@ export default {
 				center: config.center,
 				zoom: config.zoom
 			})
-			leaflet.tileLayer('https://{s}.tiles.madavi.de/{z}/{x}/{y}.png', {
+//			leaflet.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+			leaflet.tileLayer('https://maps.luftdaten.info/tiles/{z}/{x}/{y}.png', {
```
Comment (DavidLemayian) [inline src/map.vue:20]: @chegejames Same question here.

File: index.html (hunk 1 above)
Comment (ricki-z) [reply index.html:11]: @DavidLemayian We do this to limit the user access to other servers. So the user access won't be visible to Google or others (we have disabled logging on our server).
It's implemented as a apache proxy service.

File: src/map.vue (hunk 2 above)
Comment (ricki-z) [reply src/map.vue:20]: Same answer here ;-)

File: index.html (hunk 1 above)
Comment (DavidLemayian) [reply index.html:11]: Oh.. Interesting. Thanks.

**Comment by DavidLemayian** [comment]
Done by https://github.com/CodeForAfricaLabs/sensors.AFRICA-AQ-map/commit/276e9cf5bc332e0df79f58655f8cb370bde7809b

**Pull request closed without merging**